database/dbEvents.php

```diff
@@ -161,6 +162,15 @@ function fetch_events_on_date($date) {
     return $events;
 }
 
+function make_field($result_row){
+    $field = new Field(
+        null,
+        $result_row['name'],
```

Comment:
Spacing might not be necessary here (could have arguments just on one line)

Reply:
Good catch. This is formatted based on another function similar to one for a different object. Let's discuss general formatting next meeting